index.js

```diff
@@ -20,3 +20,30 @@ module.exports = fn => {
 
 	return ret;
 };
+
+const log = (fn, promise) => {
+	console.log(`Promise from ${fn.displayName} resolved in ${promise.time} ms`);
```

Comment:
👍 

Reading the docs on [displayName](https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Global_Objects/Function/displayName), it doesn't like there is a connection between `displayName` and `name`. But it probably depends on implementation because it's non-standard.